.github/README.MD

```diff
@@ -137,7 +137,7 @@ these packages if you use them directly in your project.
 
 ## 🤲 Contributing
 
-Please check out our [Contribution Guidelines](./.github/CONTRIBUTING.MD) for
+Please check out our [Contribution Guidelines](./CONTRIBUTING.MD) for
```

Comment:
So this one is tough. By changing this it's fixed when viewing the readme on Github, but will be broken when viewing it on NPM. 

Check the link on github here: https://github.com/carbon-design-system/carbon-addons-iot-react/tree/963-addon-docs

After this change, the NPM link will result in: https://github.com/carbon-design-system/carbon-addons-iot-react/blob/HEAD/CONTRIBUTING.MD which is a 404.

Reply:
Is there a way to use the absolute link in the MD?

Reply:
Yep, but even that has some side effects. It can break local links in docs. Do we point to next or master?

Reply:
fixed.

Reply:
Resolved. Adding absolute links